src/site/filters/liquid.js

```diff
@@ -315,4 +315,13 @@ module.exports = function registerFilters() {
     enabledFeatureFlags[featureFlags.FEATURE_HEALTH_SERVICE_API_ID]
       ? serviceTaxonomy.fieldHealthServiceApiId
       : serviceTaxonomy.name;
+
+  // finds if a page is a child of a certain page using the entityUrl attribute
+  // returns true or false
+  liquid.filters.isChildPageOf = (childPageEntityUrl, parentPage) =>
+    Boolean(
```

Comment:
Rather than type casting (more or less), can you use `!!` instead? (I'd use the GitHub suggestion feature, but I think it only works for one line. :man_shrugging:)

```js
liquid.filters.isChildPageOf = (childPageEntityUrl, parentPage) =>
  !!childPageEntityUrl.breadcrumb.find( ... );
```

Reply:
huh, not sure what i was thinking! i was thinking of not using the `!!` because i thought it was to check if something was false _and_ coerce into boolean form, but i guess it's just the shorthand of what i have written here :smile: